source/dunders/index.rst

```diff
@@ -11,24 +11,19 @@ Introduction
 
 Le mot dunder est un raccourci de *Double UNDERscore* et représente toutes les méthodes Python qui commencent et finissent par un double underscore (par ex. ``__init__()``).
 
-Les dunders sont des méthodes très puissantes et régulièrement utilisées en python. Ce sont des méthodes universelles que toute classe possède (un peu à l'image de la class Object dans Java).
+Les dunders sont des méthodes très puissantes et régulièrement utilisées en python. Un dunder est toujours une méthode privée appartenant à une classe. La particularité de ces méthodes et qu'elles ne sont jamais directement appelées mais toujours implicitement ou explicitement
```

Comment:
`_method` et `__method` sont des conventions oui.

> _We don't use the term "private" here, since no attribute is really private in Python (without a generally unnecessary amount of work). _
>
> https://www.python.org/dev/peps/pep-0008/#designing-for-inheritance